.github/workflows/ci.yaml

```diff
@@ -0,0 +1,84 @@
+name: Build static binaries
+
+on: push
```

Comment:
Might as well run this on `pull_request` too

Reply:
Good point. Triggers on release creation as well.